framework/py/flwr/cli/run/run.py

```diff
@@ -100,27 +101,53 @@ def run(
     try:
         if suppress_output:
             redirect_output(captured_output)
+
+        original_app_str = str(app) if app is not None else ""
```

Comment:
Do we need to introduce this variable? we already pass `app_path: Path = app` around and to `_run_with_control_api`. I think we can remove `original_app_str`